### PR TITLE
Develop express sheets

### PR DESCRIPTION
**Pull request by yuktiarora**
### What does this PR do?
Adds a Dockerfile, .dockerignore, changes the formatting of the .env file, makes the app port fixed

### CODE REVIEW

**Review by mderazon — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+COPY package*.json /usr/src/express-sheets/
+RUN npm install
+
+# Bundle app source
+COPY . /usr/src/express-sheets
+RUN npm run build
+
+EXPOSE 3000
+
+CMD npm run start
```
Comment (deepak-uttam) [inline Dockerfile]: I think we can use multistage docker build here so that we can reduce the image size.
@mderazon What do you think?

**Review by deepak-uttam — CHANGES_REQUESTED** [review]

File: .env [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
+  client_email=google-spreadsheet-access@express-sheets-trial.iam.gserviceaccount.com
+  client_id=118109966120154911750
+  auth_uri=https://accounts.google.com/o/oauth2/auth
+  token_uri=https://oauth2.googleapis.com/token
+  auth_provider_x509_cert_url=https://www.googleapis.com/oauth2/v1/certs
+  client_x509_cert_url=https://www.googleapis.com/robot/v1/metadata/x509/google-spreadsheet-access%40express-sheets-trial.iam.gserviceaccount.com
+  sheet_id=1GvlGcn-IHnacCAztYzIENNIZs4bc_Ys4zVcEM94LPFU
+  update_interval_time=30000
```
Comment (deepak-uttam) [inline .env]: We should not commit this in git as this is going to be public repository.

File: Dockerfile (hunk 1 above)
Comment (mderazon) [reply Dockerfile]: I think we could

File: .env (hunk 2 above)
Comment (mderazon) [reply .env]: this is a sample spreadsheet created for this purpose... but maybe the secret keys should be passed on from env, I agree

File: Dockerfile (hunk 1 above)
Comment (yuktiarora) [reply Dockerfile]: @deepak-uttam 
@mderazon 
Please review now, I have created a multistage Dockerfile

**Pull request merged**